tests/foreman/cli/test_activationkey.py

```diff
@@ -628,41 +585,31 @@ def test_positive_update_activation_key_1(self, name):
 
         """
         try:
-            activation_key = self._make_activation_key({
-                u'organization-id': self.org['id'],
-            })
+            activation_key = self._make_activation_key()
         except CLIFactoryError as err:
             self.fail(err)
+        for name in valid_data_list():
+            with self.subTest(name):
+                result = ActivationKey.update({
+                    u'id': activation_key['id'],
+                    u'new-name': name,
+                    u'organization-id': self.org['id'],
+                })
+                self.assertEqual(result.return_code, 0, 'Failed updating ak')
```

Comment:
for these messages, would it be better to remove our generic "failed ____" message that suppresses the assertion error? We already know that the update failed because of the test's name. This way we can get more useful messages. This might not be in the scope of this PR, but i'll let you decide that.